sam_stonefish_sim/scripts/bringup.sh

```diff
@@ -48,11 +48,18 @@ case "$SCENARIO" in
 		LONGITUDE=15.44264
 		;;
 	"algae_world")
+        # Algae farm (2 long ropes)
+		UTM_ZONE=33
+		UTM_BAND=T
+		LATITUDE=58.250833
+		LONGITUDE=11.450283
+		;;
+	"algae_world_small")
 		# Algae farm
 		UTM_ZONE=33
 		UTM_BAND=T
```

Comment:
These need to change to match the UTM zone and band of Kristineberg. There should be an example in the sam drivers bringup script

Reply:
This is what it says in that script:
```
# Kristineberg
UTM_ZONE=32
UTM_BAND=V
```

Reply:
Thanks for pointing out! I just changed the utm accordingly.